### PR TITLE
Send activity updates to server

### PR DESCRIPTION
**Pull request by borisyankov**
Implements #50 

Send activity updates:
* on activity like new narrow or scroll messages (at most once per minute)
* on input being typed (at most once per 5 seconds)

### CODE REVIEW

**Comment by smarx** [comment]
*Automated message from Dropbox CLA bot*

@borisyankov, it looks like you've already signed the Dropbox CLA. Thanks!

File: src/account-info/AccountDetailsScreen.js [hunk 1]
```diff
@@ -28,7 +28,7 @@ class AccountDetailsScreen extends Component {
     const fullAvatarUrl = getFullUrl(avatarUrl, auth.realm);
 
     return (
-      <Screen title="Account details">
+      <Screen title="Account details" padded>
```
Comment (neerajwahi) [inline src/account-info/AccountDetailsScreen.js]: I don't think `padded` does anything. You'll need to wrap with a view for padding

**Comment by neerajwahi** [comment]
Also, please make sure CI passes before merging

**Comment by borisyankov** [comment]
Removed `padded` and rebased.

**Pull request merged**